.bluemix/pipeline_tekton_cf.yml

```diff
@@ -7,7 +7,7 @@ inputs:
   - type: git
     branch: ${PIPELINE_REPO_BRANCH}
     service: ${PIPELINE_REPO}
-    path: tasks/skit-verification
+    path: tasks/skit-alert-and-verification
```

Comment:
I renamed this task to alert and verification but it should really be verification and registration or skit-verify-alert-register something like that